CHANGELOG.md

```diff
@@ -12,6 +12,134 @@ The format is inspired by [Keep a Changelog].
 
 <!-- TOWNCRIER -->
 
+## 1.8.0 (2023-10-18)
+
+### Process Changes
+
+- Add Change Log and the Change Log fragments process for assembling it
+  ([#1645](https://github.com/oasisprotocol/oasis-wallet-web/issues/1645))
+
+  This follows the same Change Log fragments process as is used by [Oasis Core].
+
+  For more details, see [Change Log fragments].
+
+  [Oasis Core]: https://github.com/oasisprotocol/oasis-core
+  [Change Log fragments]: .changelog/README.md
+
+### Features
+
+- Enable ParaTimes transfers
+  ([#1636](https://github.com/oasisprotocol/oasis-wallet-web/issues/1636))
+- Address Book
+  ([#1635](https://github.com/oasisprotocol/oasis-wallet-web/issues/1635),
+  [#1657](https://github.com/oasisprotocol/oasis-wallet-web/issues/1657),
+  [#1661](https://github.com/oasisprotocol/oasis-wallet-web/issues/1661),
+  [#1668](https://github.com/oasisprotocol/oasis-wallet-web/issues/1668),
+  [#1674](https://github.com/oasisprotocol/oasis-wallet-web/issues/1674),
+  [#1696](https://github.com/oasisprotocol/oasis-wallet-web/issues/1696))
+- Password Change
+  ([#1675](https://github.com/oasisprotocol/oasis-wallet-web/issues/1675))
+- Settings Page
+  ([#1666](https://github.com/oasisprotocol/oasis-wallet-web/issues/1666),
+  [#1630](https://github.com/oasisprotocol/oasis-wallet-web/issues/1630),
+  [#1655](https://github.com/oasisprotocol/oasis-wallet-web/issues/1655),
+  [#1713](https://github.com/oasisprotocol/oasis-wallet-web/issues/1713))
+- Delete profile modal
+  ([#1641](https://github.com/oasisprotocol/oasis-wallet-web/issues/1641))
+
+### Bug Fixes and Improvements
+
+- Export private key re-design
+  ([#1704](https://github.com/oasisprotocol/oasis-wallet-web/issues/1704),
+  [#1720](https://github.com/oasisprotocol/oasis-wallet-web/issues/1720))
+- Adjust gas cost for Sapphire and Emerald
+  ([#1712](https://github.com/oasisprotocol/oasis-wallet-web/issues/1712))
+- Fix refreshing balances on Home page after switching network
+  ([#1691](https://github.com/oasisprotocol/oasis-wallet-web/issues/1691))
+- Fix continuously refreshing balances in wallet slice
+  ([#1692](https://github.com/oasisprotocol/oasis-wallet-web/issues/1692))
+- Fix scrolling modals on phone by replacing 100vh with 100dvh
+  ([#1702](https://github.com/oasisprotocol/oasis-wallet-web/issues/1702))
+
+### Internal Changes
+
+- Miscellaneous improvements
+  ([#1614](https://github.com/oasisprotocol/oasis-wallet-web/issues/1614),
+  [#1623](https://github.com/oasisprotocol/oasis-wallet-web/issues/1623),
+  [#1624](https://github.com/oasisprotocol/oasis-wallet-web/issues/1624),
+  [#1621](https://github.com/oasisprotocol/oasis-wallet-web/issues/1621),
+  [#1637](https://github.com/oasisprotocol/oasis-wallet-web/issues/1637),
+  [#1639](https://github.com/oasisprotocol/oasis-wallet-web/issues/1639),
+  [#1640](https://github.com/oasisprotocol/oasis-wallet-web/issues/1640))
+- Dependency updates
```

Comment:
Something to consider for the future:

Should we introduce a separate section and Change Log fragment extension just for Dependency updates?

I think there are many more dependency updates here than in some average Go project.

Reply:
Filled https://github.com/oasisprotocol/oasis-wallet-web/issues/1730